next-app/src/components/features/review/ReviewForm/ReviewForm.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import RatingInput from "@/components/shared/RatingInput/RatingInput";
+import { Box, Button, Heading, Stack, Textarea } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+interface ReviewFormProps {
+  addShowReview: (comment: string, rating: number) => void;
+}
+
+const ReviewForm: React.FC<ReviewFormProps> = ({ addShowReview }) => {
+  const [comment, setComment] = useState("");
+  const [rating, setRating] = useState(5);
+
+  const [commentIsInvalid, setCommentIsInvalid] = useState(false);
+
+  const isValidComment = (comment: string): boolean => {
+    if (!comment) return false;
+
+    return true;
+  };
+  const isValidRating = (rating: number): boolean => {
+    if (!(rating >= 1 && rating <= 5)) return false;
+
+    return true;
+  };
+
+  // Validate form, set invalid states and return true if invalid
+  const validateForm = (): boolean => {
+    let formIsInvalid = false;
+    if (!isValidComment(comment)) {
+      setCommentIsInvalid(true);
+      formIsInvalid = true;
+    }
+    if (!isValidRating(rating)) {
+      formIsInvalid = true;
+    }
+
+    return formIsInvalid;
+  };
+
+  const resetForm = () => {
+    setComment("");
+    setRating(5);
+    setCommentIsInvalid(false);
+  };
+
+  return (
+    <Stack alignItems="flex-start">
```

Comment:
Prošli smo ovo na konzultacijama pa evo mali komentar samo:
S obzirom da smo komponentu nazvali ReviewForm očekivao bih da je ovo stv. forma. tj form element i tako i hendlao podatke.
Možeš ovdje dodati as="form" i onda hendlaš onSubmit event (maknes onClick sa button-a...), vjerujem da ti je jasan pristup :)
Semantički je točnije a i u tom smjeru radi i react-hook-form kojeg ćemo ubuduće koristiti :)